apps/server/src/modules/video-conference/uc/video-conference.uc.spec.ts

```diff
@@ -200,7 +200,7 @@ describe('VideoConferenceUc', () => {
 		schoolUc.hasFeature.mockResolvedValue(true);
 		courseRepo.findById.mockResolvedValue(course);
 		calendarService.findEvent.mockResolvedValue(event);
-		authorizationService.hasPermissionsByReferences.mockReturnValue(userPermissions);
+		authorizationService.hasPermissionByReferences.mockReturnValue(Promise.resolve(true));
```

Comment:
authorizationService.hasPermissionByReferences.mockResolveValueOnce(true);

This is miss used in the complet test file.